package.json

```diff
@@ -59,13 +59,13 @@
     "ajv": "^4.11.4",
     "avvio": "^0.3.0",
     "body": "^5.1.0",
-    "fast-json-stringify": "^0.10.3",
-    "fast-safe-stringify": "^1.1.6",
+    "fast-json-stringify": "^0.10.4",
+    "fast-safe-stringify": "^1.1.11",
     "fastseries": "^1.7.2",
     "middie": "^0.1.1",
     "pathname-match": "^1.2.0",
     "pino-http": "^2.4.2",
     "pump": "^1.0.2",
-    "wayfarer": "^6.4.1"
+    "wayfarer": "^6.5.0"
```

Comment:
do you update those manually all the time, or you use some script? It's not needed, the new versions are picked by the semver range.